8/debian/buster/Dockerfile

```diff
@@ -50,6 +50,12 @@ WORKDIR "${JENKINS_AGENT_HOME}"
 
 COPY setup-sshd /usr/local/bin/setup-sshd
 
+RUN echo -e "JAVA_HOME=${JAVA_HOME}\nexport JAVA_VERSION=${JAVA_VERSION}\nexport JENKINS_AGENT_HOME=${JENKINS_AGENT_HOME}" >> /etc/environment
```

Comment:
I ran this container and this is what's in my file:

```
root@fb31fe775250:/home/jenkins# cat /etc/environment
-e JAVA_HOME=/usr/local/openjdk-8
export JAVA_VERSION=8u292
export JENKINS_AGENT_HOME=/home/jenkins
```

the java_home doesn't look right